backend/src/main/java/com/cruru/club/exception/ClubNameCharacterException.java

```diff
@@ -0,0 +1,12 @@
+package com.cruru.club.exception;
+
+import com.cruru.advice.BadRequestException;
+
+public class ClubNameCharacterException extends BadRequestException {
+
+    public static final String MESSAGE = "동아리 이름에 \\ 혹은 |를 포함하고 있습니다. 입력된 이름: %s";
+
+    public ClubNameCharacterException(String invalidName) {
+        super(String.format(MESSAGE, invalidName));
```

Comment:
이렇게 메세지를 주는 것보다 Invaild한 문자가 무엇인지만 보여주는것이 어떨까요?

```suggestion
        super(String.format(MESSAGE, invalidChar));
```

Reply:
정규 표현식을 통해 예외를 잡고 있어 어떤 문자가 문제인 지 확인하는 방법을 모르겠....습니다.